clappr/clappr-ios
# Rename DoubleTap to Jump

### Goal

To keep similarity between all Clapprs project, we decided to change _Double Tap Plugin_ to _Jump Plugin_

### How to test
- Run all Clappr tests
- Check if We forgot to rename something
- Run the application and use the Jump Feature
